<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>coldialog</class>
 <widget class="QDialog" name="coldialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>217</width>
    <height>110</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>217</width>
    <height>110</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>217</width>
    <height>110</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>添加方式选择</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="2">
      <widget class="QRadioButton" name="radioButton_beforecol">
       <property name="text">
        <string>选中列之前</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QRadioButton" name="radioButton_aftercol">
       <property name="text">
        <string>选中列之后</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_ok">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_ok</sender>
   <signal>clicked()</signal>
   <receiver>coldialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>47</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_cancel</sender>
   <signal>clicked()</signal>
   <receiver>coldialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>132</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
